### DISCUSSION TITLE
Possibility to manually trigger a fetching

### DISCUSSION THREAD

**Discussion by drueppela** [mampfes/hacs_waste_collection_schedule#2011] [Q&A] [answered]
After a restore, there are no waste collection schedules.

![image](https://github.com/mampfes/hacs_waste_collection_schedule/assets/9786634/3774aada-8891-4fb1-b9fb-7b695b7f91ea)

I either have to wait for the next configured fetching or I could change the fetching time as a one-timer and then change it back.

Is there a possibility to manually trigger a fetching?

**Comment by 5ila5** [comment]
The integration should fetch by itself after a restart. You can activate the `waste_collection_schedule.fetch_data` service in the developer tools section (but I never used it, so I'm not 100% sure this actually works)

**Reply by drueppela** [reply]
@5ila5 : When i search for "waste" i only find these entities:

![image](https://github.com/mampfes/hacs_waste_collection_schedule/assets/9786634/385678c0-ecfd-46fb-8ead-1cbf176a88ce)

None of these trigger fetching the waste schedules.

**Reply by 5ila5** [reply, marked as answer]
You can trigger the sevice in developer tools -> services tabs
![grafik](https://github.com/mampfes/hacs_waste_collection_schedule/assets/38183212/7f288886-a769-4886-8466-08e025d5d8bb)

**Reply by drueppela** [reply]
You just need to do it right to make it work. Thank you!

**Reply by drueppela** [reply]
Example Button Card configuration:

```show_name: true
show_icon: false
type: button
tap_action:
  action: call-service
  service: waste_collection_schedule.fetch_data
  target: {}
icon: ''
show_state: true
name: 'Name of the button'```
